piqueserver/game_modes/infiltration.py

```diff
@@ -78,8 +78,8 @@ def score(self):
         intel_capture.winning = winning
         self.protocol.send_contained(intel_capture, save=True)
         if winning:
-            self.team.initialize()
-            self.team.other.initialize()
+            self.team = self.team(*self.team.get_init_values())
+            self.team.other = self.team(*self.team.other.get_init_values())
```

Comment:
We could modify the reset_teams method to allow using it here as well. I'm thinking it's probably not gonna make the code better but its worth considering.